<ui version="4.0" >
 <class>lp_ladspa_knob_float</class>
 <widget class="QWidget" name="lp_ladspa_knob_float" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>136</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="txt_name" >
     <property name="maximumSize" >
      <size>
       <width>16000</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QwtKnob" name="kb_val" >
     <property name="maximumSize" >
      <size>
       <width>16000</width>
       <height>16000</height>
      </size>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_def_val" >
     <property name="maximumSize" >
      <size>
       <width>16000</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="sp_val" >
     <property name="maximumSize" >
      <size>
       <width>16000</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
